### DISCUSSION TITLE
How to clear & reimport PCAPs for analysis?

### DISCUSSION THREAD

**Discussion by srinagars** [Security-Onion-Solutions/securityonion#2070] [Unsupported Versions] [answered]
How can I clear previously imported PCAPs and then re-import them? So alerts pop up again for each PCAP imported

**Comment by weslambert** [comment, marked as answer]
You could try something like `sudo rm -rf /nsm/import/* && sudo so-elastic-clear -y` if you are referring to an installation where you have only imported PCAP, and have not been recording live data.  Otherwise, instead of `so-elastic-clear`, you would need to remove the `so-import` indices from Elasticsearch.
